modules/ROOT/pages/setup.adoc

```diff
@@ -35,17 +35,17 @@ If you are unsure about the Mule runtime capacity required to handle your B2B tr
 == Environment Configuration
 
 You'll need to set up and configure your environment before you begin using Partner Manager.
-Please refer to Anypoint platform documentation to learn more about https://docs.mulesoft.com/access-management/environments[Configuring your Anypoint platform environments].
+Refer to the Anypoint platform documentation to learn more about https://docs.mulesoft.com/access-management/environments[Configuring your Anypoint platform environments].
```

Comment:
Suggest omitting, because this is a big topic and your audience needs only the items detailed next.
Should you keep it:
```suggestion
Refer to the Anypoint Platform documentation to learn more about https://docs.mulesoft.com/access-management/environments[configuring your Anypoint platform environments].
```

Reply:
?